CHANGELOG.md

```diff
@@ -4,23 +4,187 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## develop (unreleased) 2020-12-08
-
-### Bug Fixes
-- `fuel_used_gal` output for `Generator` was incorrect for scenarios with `time_steps_per_hour != 1`
-- `check_common_outputs` was not testing that outputs of type `list` are both `list`s in the expected and calculated outputs
-
-### Changes
-- changed `test_flexible_time_steps` to include an outage
-- changed check in `validators.py` from "outage_start_hour and outage_end_hour cannot be the same" to "outage_start_hour must be less than outage_end_hour"
-- new inputs `outage_start_time_step` and `outage_end_time_step` to replace deprecated `outage_start_hour` and `outage_end_hour`. The latter are used as time step indices in the code, so for sub-hourly problems they do not have hourly units. For now `outage_start_hour` and `outage_end_hour` are kept in place to preserve backwards-compatibility. Also note that the new inputs are not zero-indexed.
-- new output `bau_sustained_time_steps` to replace deprecated `sustain_hours` (also not deprecated yet but warning is now in response).
-- new `dvProductionToCurtail` in reopt.jl for all techs in all time steps (
-    - was previously construed with dvProductionToGrid for the third sales tier, which is meant for selling energy back to the grid beyond the annual load kWh constraint.
-- new inputs for all Techs: `can_net_meter`, `can_wholesale`, `can_export_beyond_site_load`, `can_curtail`
+## Guidelines
+
+Use **bold** markup for field and model names (i.e. **outage_start_time_step**)
+
+Use `code` markup for  REopt Lite-specific file names, classes and endpoints (i.e `reo/validators.py`)
+
+Use _italic_ for code terms (i.e. _list_)
+
+Prepend change with tag(s) directing where it is in the repository: 
+    
+    
+`reo`,`proforma`,`resilience_stats`,`*.jl`,`REopt_Lite_API`
+
+Classify the change according to the following categories:
+    
+    ### Major Updates
+    ### Minor Updates
+    ##### Added
+    ##### Changed
+    ##### Fixed
+    ##### Deprecated
+    ##### Removed
+    ### Patches
+
+## develop (unreleased)
+### Major Updates
+##### Added
```

Comment:
@NLaws Are these Major additions, or did the Minor heading just get missed after the Major heading?

Reply:
@Bill-Becker I thought it made sense to break up major changes by Added, Changed , Fixed, Deprecated, Removed since these are recommended categories on https://keepachangelog.com/en/1.0.0/. Patches seemed like they would always be minor fixes and upgrades so not sure we wanted to go into as much detail there.

Does that make sense to you? There's no one way to do this, so open to feedback/suggestions